src/dashboard-modal.scss

```diff
@@ -0,0 +1,44 @@
+@import "color";
+
+$inputWidth: 75%;
+
+.dashboardModal {
```

Comment:
nit: please use kebab-case for CSS classes to be consistent with the existing CSS